Sources/Logging/Loggable.swift

```diff
@@ -170,8 +170,30 @@ extension MXDiagnosticPayload {
 @available(iOS 14.0, *)
 extension MXCrashDiagnostic {
     var logDescription: String {
-        // swiftlint:disable:next line_length
-        return "💥 Reason: \(terminationReason ?? ""), Type: \(exceptionType?.stringValue ?? ""), Code: \(exceptionCode?.stringValue ?? ""), Signal: \(signal?.stringValue ?? ""), OS: \(metaData.osVersion), Build: \(metaData.applicationBuildVersion)"
+        return """
+        💥  Reason: \(terminationReason ?? "")
+            Type: \(exceptionType?.stringValue ?? "")
+            Code: \(exceptionCode?.stringValue ?? "")
+            Signal: \(signal?.stringValue ?? "")
+            OS: \(metaData.osVersion)
+            Build: \(metaData.applicationBuildVersion)
+
+        \(callStackTree.logDescription)
+        """
+    }
+}
+
+@available(iOS 14.0, *)
+extension MXCallStackTree {
+    var logDescription: String {
+        let jsonData = jsonRepresentation()
+
+        guard let object = try? JSONSerialization.jsonObject(with: jsonData, options: []),
+              let data = try? JSONSerialization.data(withJSONObject: object, options: [.prettyPrinted]) else {
+            return ""
```

Comment:
Might not be a bad idea to print information about the error that caused this to return an empty string?

Reply:
That error would end up as noise inside the Diagnostics Reports, do you still think that's useful?

Reply:
And `""` wouldn't? I think it'd be useful — having an error as to why this couldn't converted to / from JSON would not hurt.